### ISSUE TITLE
Tour Mode Choice Coefficients

### ISSUE THREAD

**Issue by danielsclint** [wsp-sag/client_arc_activitysim#16]
Using the [Mode Choice UEC](https://github.com/wsp-sag/ARC_Model/blob/master/ctrampModels/ModeChoice.xls) as guide, fill in all of the tour mode choice coefficients into the tour_mode_choice_coeff.csv table.

Only focus on variables in the Excel UEC that start with 'c_'. Some may require formulas in lieu of straight numerical values. You can use the [MTC Travel Model 1 tour_mode_choice_coeff.csv](https://github.com/ActivitySim/activitysim/blob/master/example/configs/tour_mode_choice_coeffs.csv) as a guide as well for formatting.

The table will look something like this....

|Expression|Work    |Univeristy|School|NonMandatory|AtWork  |
|---            |--------:|-----------:|-------:|----------------:|---------:| 
|c_ivt         |-0.0247|-0.0247     |-0.0064|-0.02142         |-0.02142|
|c_ovt        |0          |-0.0457     |-0.0127|-0.05432         |-0.05432|
|...              |....         |....              |...         |...                     |...           |

**Comment by chryssac** [comment]
@danielsclint: I have a question. Some of the values are the result of multiplying a previous cell by a constant. Do I translate like that with the formula on the spreadsheet or do I use the value? For example, c_firstWaitShort in work is =E$15*2 which evaluates to -0.0494

**Comment by chryssac** [comment]
I kept the formulas. I checked in the updated file under a new branch, mode_choice_coeffs

**Comment by danielsclint** [comment]
Good catch, I would advocate for keeping the formulas. In the current check I only see the values. For example, in the work column I would put:

c_knrTime = c_ivt * 2.5

This keeps the reminder of the ratio for future editing and calibration.

**Comment by danielsclint** [comment]
Also, for the _hh_income stuff on lines 16-20... move that to the annotate_hh.csv.

**Comment by chryssac** [comment]
I checked in the updated version

**Comment by chryssac** [comment]
@danielsclint I took the reindexing out and used the households.hincp field instead to define the different income brackets. I also moved these to the annotate_households.csv file. But now it can't find those definitions by the time in gets to reading the mode choice coefficients.... Still looking into it but I thought the annotate_households happens before the coefficients and normally it should see these definitions...

**Comment by chryssac** [comment]
I have checked my progress so far on the tour_mode_choice.csv file. Per your recommendation, I have added comments to the ones that will need further investigation.
